tests/unit/repo/experiments/queue/test_local.py

```diff
@@ -0,0 +1,37 @@
+from celery import shared_task
+
+
+def test_shutdown_no_tasks(test_queue, mocker):
+    shutdown_spy = mocker.spy(test_queue.celery.control, "shutdown")
+    test_queue.shutdown()
+    shutdown_spy.assert_called_once()
+
+
+@shared_task
+def _foo():
+    return "foo"
```

Comment:
note that the pytest-celery docs are inaccurate, tasks must be defined at the module scope or else the threaded test worker fixture won't pick them up

(test inlined tasks will be registered to the celery app after the worker has already started, and the worker will not reload the task list)